pages/your_tasks/data_analysis.md

```diff
@@ -71,7 +71,7 @@ Conda works by making a nested folder containing the traditional UNIX directory
 * Tips and tricks to navigate the landscape of software package management solutions:
     * If you need multiple tools/programming languages, but your machines have different OS types or versions, list packages in a Conda `environment.yml`.
```

Comment:
Suggested revision: _"If you need multiple tools or programming languages for a machine that has a different OS type or version, you can list relevant packages in Conda environment via your `environment.yaml` file."_ ?

If you want to tell a reader about fetching a list of packages in conda environment, you can mention `conda list -n <my_env>`.

Reply:
I like more the use of "Manage the software you need in an OS independent way by listing all relevant packages in the Conda environment via your environment.yaml file.

What do you think?

Reply:
Sounds good!